cmd/docker-machine-driver-kvm/Makefile

```diff
@@ -1,4 +1,4 @@
 default: build
 
 build:
-	GOGC=off go build -i -o docker-machine-driver-kvm
+	GOGC=off go build -tags libvirt.1.2.14 -i -o docker-machine-driver-kvm
```

Comment:
Unfortunately I don't have a physical ubuntu 14.04 box (or other older LTS distro) to try this out at the moment.  Was this alone sufficient to have this all hold together where it both compiles and doesn't do something nasty at runtime when the underlying API isn't present?  Maybe enough time has passed that we can ignore the older LTS distros, but I was hoping to keep this compatible for a while if possible.